src/darwin/Framework/CHIP/MTROTAProviderDelegateBridge.h

```diff
@@ -65,6 +67,7 @@ class MTROTAProviderDelegateBridge : public chip::app::Clusters::OTAProviderDele
     static void ConvertToNotifyUpdateAppliedParams(
         const chip::app::Clusters::OtaSoftwareUpdateProvider::Commands::NotifyUpdateApplied::DecodableType & commandData,
         MTROTASoftwareUpdateProviderClusterNotifyUpdateAppliedParams * commandParams);
+    static void HandleBdxInitReceivedTimeoutExpired(chip::System::Layer * systemLayer, void * state);
```

Comment:
Again, this method is not actually implemented and should not be declared here.

Reply:
done